src/xeus_sqlite_interpreter.cpp

```diff
@@ -352,7 +352,7 @@ nl::json interpreter::execute_request_impl(int execution_counter,
             html_table << "</table>";
         }
 
-        if (result.size())
+        if (result.size() > 2)
```

Comment:
This looks like a weak test to me. Why did you choose `2`? Will it not change eventually?